### PR TITLE
feat(console): add spring boot integration guide

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Add spring boot integration guide

image

image

## Testing

test locally

## Checklist

- [x] `.changeset`
- [ ] ~unit tests~
- [ ] ~integration tests~
- [ ] ~necessary TSDoc comments~

### CODE REVIEW

**Review by charIeszhao — APPROVED** [review]

File: .changeset/soft-stingrays-beam.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@logto/console": patch
+---
+
+Add new jave spring boot web integration guide to the application creation page.
```
Comment (gao-sun) [inline .changeset/soft-stingrays-beam.md:5]:
```suggestion
add Java spring boot web integration guide to the application creation page
```

File: packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts [hunk 2]
```diff
@@ -3,10 +3,13 @@ import { ApplicationType } from '@logto/schemas';
 import { type GuideMetadata } from '../types';
 
 const metadata: Readonly<GuideMetadata> = Object.freeze({
-  name: 'Java Web',
-  description:
-    'Java Web is a web framework for Java that enables developers to build secure, fast, and scalable server applications with the Java programming language.',
+  name: 'Java Spring Boot Web',
+  description: 'Integrate Logto with a Java Spring Boot web application.',
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts:7]: no need to change the description?

Comment (gao-sun) [reply packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts:7]: yes, i mean this description should be relevant to the Spring Boot framework

Comment (simeng-li) [reply packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts]: updated

File: packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { ApplicationType } from '@logto/schemas';
+
+import { type GuideMetadata } from '../types';
+
+const metadata: Readonly<GuideMetadata> = Object.freeze({
+  name: 'Java Spring Boot Web',
+  description:
+    'Spring Boot is a web framework for Java that enables developers to build secure, fast, and scalable server applications with the Java programming language. ',
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts:8]:
```suggestion
    'Spring Boot is a web framework for Java that enables developers to build secure, fast, and scalable server applications with the Java programming language.',
```

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts (hunk 3 above)
Comment (simeng-li) [reply packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts]: updated

**Pull request merged**